charts/seed-bootstrap/charts/loki/templates/statefulset.yaml

```diff
@@ -42,6 +42,21 @@ spec:
         fsGroup: 10001
         fsGroupChangePolicy: "OnRootMismatch"
       priorityClassName: {{ .Values.priorityClassName }}
+      initContainers:
+      - command:
+        - bash
+        - -c
+        - tune2fs -O large_dir $(lsblk -o KNAME,MOUNTPOINTS | gawk 'NR>1 {if($2=="/data") {print "/dev/"$1}}')
```

Comment:
It seems that the local setup does not like this init container:
```
2023-03-16T13:52:45.744931265Z stdout F tune2fs 1.46.6 (1-Feb-2023)
2023-03-16T13:52:45.744966915Z stderr F Usage: tune2fs [-c max_mounts_count] [-e errors_behavior] [-f] [-g group]
2023-03-16T13:52:45.744990325Z stderr F 	[-i interval[d|m|w]] [-j] [-J journal_options] [-l]
2023-03-16T13:52:45.744993165Z stderr F 	[-m reserved_blocks_percent] [-o [^]mount_options[,...]]
2023-03-16T13:52:45.744995145Z stderr F 	[-r reserved_blocks_count] [-u user] [-C mount_count]
2023-03-16T13:52:45.744997235Z stderr F 	[-L volume_label] [-M last_mounted_dir]
2023-03-16T13:52:45.744998975Z stderr F 	[-O [^]feature[,...]] [-Q quota_options]
2023-03-16T13:52:45.745000735Z stderr F 	[-E extended-option[,...]] [-T last_check_time] [-U UUID]
2023-03-16T13:52:45.745002755Z stderr F 	[-I new_inode_size] [-z undo_file] device
```

Most probably the tests fail because of it.

Reply:
the `lsblk` give totally different output on the local set-up
```
KNAME MOUNTPOINTS
nbd0
nbd1
nbd2
nbd3
nbd4
nbd5
nbd6
nbd7
vda
vda1  /etc/resolv.conf
      /etc/hostname
      /dev/termination-log
      /etc/hosts
      /data
nbd8
nbd9
nbd10
nbd11
nbd12
nbd13
nbd14
```

Can I bypass this `tune2fs` command success with `|| true`?

Reply:
With `mount` command, we are able to get the device name on each line.
It is easier to extract the device this way even when there are multiple mounting points.